Specs/DateTime/English/DateTimeModel.json

```diff
@@ -23962,5 +23962,304 @@
         }
       }
     ]
+  },
+  {
+    "Input": "schedule me a 1:1 meeting",
+    "Comment": "Taskmode supress 1:1 from getting categorised as time.",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "javascript",
+    "Results": [
+      {
+        "Text": "1:1",
+        "Start": 14,
+        "End": 16,
+        "TypeName": "datetimeV2.time",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "T01:01",
+              "type": "time",
+              "value": "01:01:00"
+            },
+            {
+              "timex": "T13:01",
+              "type": "time",
+              "value": "13:01:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
```

Comment:
This one should be modified to remove the 1:1 extraction once functionality is corrected.

Reply:
Changes integrated